src/app/repos/viewed/page.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import Loader from "@/components/Loader";
+import Navbar from "@/components/Nav";
+import Repo from "@/components/Repo";
+import type { GithubData } from "@/components/Repos";
+import { useEffect, useState } from "react";
+
+export default function Page() {
+  const [data, setData] = useState<GithubData[]>([]);
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      const storedRepos = localStorage.getItem("ttViewedRepos");
+      const parsedData = storedRepos ? JSON.parse(storedRepos) : [];
+      setData(parsedData);
+    }
+  }, []);
+
+  return (
+    <main>
+      <Navbar />
+      {data ? (
```

Comment:
Add search filtration here also

Reply:
✅ - also added in date filter to filter based on date.